src/appleseed/renderer/modeling/surfaceshader/physicalsurfaceshader.cpp

```diff
@@ -123,6 +126,15 @@ namespace
             if (m_lighting_samples > 1)
                 radiance /= static_cast<float>(m_lighting_samples);
 
+            if (const Material* material = shading_point.get_material())
```

Comment:
I wonder if this wouldn't be best done in `ShadingEngine`.

Reply:
I thought about that, but in ShadingEngine I don't have easy access to the AOVs.